povm_toolbox/quantum_info/multi_qubit_frame.py

```diff
@@ -32,8 +35,14 @@
 
 from .base import BaseFrame
 
+LabelMultiQubitT = TypeVar("LabelMultiQubitT", int, tuple[int, ...])
+"""Each operator in the multiqubit frame is identified by a label.
```

Comment:
```suggestion
"""Each operator in the multi-qubit frame is identified by a label.
```
Either this, or we actually write:
> in the frame spanning multiple qubits